app/views/articles/show.html.erb

```diff
@@ -1,17 +1,32 @@
-<h1><%= @article.title %></h1>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Blog</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  <head>
 
-<p><%= @article.body %></p>
+  <body>
 
-<ul>
-  <li><%= link_to "Edit", edit_article_path(@article) %></li>
-  <li><%= link_to "Destroy", article_path(@article), data: {
-                    turbo_method: :delete,
-                    turbo_confirm: "Are you sure?"
-                  } %></li>
-</ul>
+    <h1><%= @article.title %></h1>
+    <p><%= @article.body %></p>
 
-<h2>Comments</h2>
-<%= render @article.comments %>
+    <% if user_signed_in?  %>
+      <% if @article.owner == current_user.id %>
```

Comment:
Could we move this condition to a method and then call the method on the view? Just like you do with `user_signed_in?`

Reply:
Yes, this condition could be moved to a method. I will add this in the next pull request

Reply:
The best thing would be to add that change in this PR, so I can check the changes, the same for the comment https://github.com/miguel0577/blog/pull/12#discussion_r991333350